doc/content/getting_started/installation.md

```diff
@@ -59,22 +59,52 @@ To compile FENIX, first make sure that the conda MOOSE environment is activated
 conda activate moose
 ```
 
-Then navigate to the FENIX clone directory and download the MOOSE and TMAP8 submodules:
+Then navigate to the FENIX clone directory and download the MOOSE, TMAP8, and Cardinal submodules:
```

Comment:
The documentation needs to be updated to have a `Tips` message to explain how they can build FENIX without Cardinal if they desire.